pages/mesosphere/dcos/1.13/release-notes/1.13.8/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 1.13.8
+title: Release notes for 1.13.8
+menuWeight: 2
+excerpt: Release notes for DC/OS 1.13.8, including Open Source attribution, and version policy.
+---
+DC/OS&trade; 1.13.8 was released on 19 March 2020.
+
+*NOTE:* The DC/OS 1.13.8 patch has been deprecated in favor of the 1.13.9 patch. D2iQ highly recommends downloading the 1.13.9 patch here instead. 
```

Comment:
I would phrase this as "DC/OS 1.13.8 has a critical bug in its networking stack. D2iQ highly recommends to not install/upgrade to this version if possible and install/upgrade to DC/OS 1.13.9+ (link to 1.13.9 release notes here) instead."

Can this note text be in red color or something to draw more attention?

Reply:
@vinodkone I took some of your suggestions, but not all. :) Check it out. 

![64862234-9CA8-40FC-9C78-B0C59581DD4D](https://user-images.githubusercontent.com/43682027/80762988-2d561380-8b03-11ea-810e-4b0c969f2886.jpeg)

Reply:
I would call it a "release" instead of "patch". Also instead of "download and install" I would say "install or upgrade"

Reply:
agree with release to upgrade. Not a fan of getting rid of download since that's the button name we're trying to get them NOT to click.